src/backend/crew_ai/agents.py

```
@@ -1,21 +1,45 @@
+    The cleaning is transparent - the LLM behaves exactly the same but with
+    robust parsing that handles real-world LLM behavior.
+
+    Args:
+        model_provider: "local" for Ollama, "online" for Gemini
+        model_name: Model identifier (e.g., "llama3.1", "gemini-2.5-flash")
+
+    Returns:
+        Cleaned LLM wrapper instance that automatically fixes formatting issues
+    """
+    logger.info(
+        f"🧹 Initializing cleaned LLM wrapper for {model_provider}/{model_name}")
```

Ruff (0.15.5)

[warning] Logging statement uses f-string

(G004)